android/src/main/java/com/auth0/react/A0Auth0Module.java

```diff
@@ -140,7 +140,13 @@ public String getName() {
     }
 
     @ReactMethod
-    public void webAuth(String scheme, String state, String nonce, String audience, String scope, String connection, int maxAge, String organization, String invitationUrl, int leeway, Promise promise) {
+    public void webAuth(String scheme, String state, String nonce, String audience, String scope, String connection, int maxAge, String organization, String invitationUrl, int leeway, boolean ephemeralSession, ReadableMap additionalParameters, Promise promise) {
```

Comment:
Is `ephemeralSession` needed in the method signature for the Android platform?

Reply:
The method signature for iOS and Android has to be same. So we are adding the missing parameters as well.